janitor/functions.py

```diff
@@ -25,6 +25,71 @@
 )
 
 
+@pf.register_dataframe_method
+def move(
+    df: pd.DataFrame,
+    source: Union[int, str],
+    target: Union[int, str],
+    position: str = "before",
+    axis: int = 0,
+) -> pd.DataFrame:
+    """
+     Move column or row to a position adjacent to another column or row in dataframe.
+
+     :param df: The pandas Dataframe object.
+     :param int or str source: column or row to move
+     :param str target: column or row to move adjacent to
+     :param str position: Specifies whether the series is moved to before or after the adjacent series. Values can be either 'before' or 'after'; defaults to 'before'.
```

Comment:
The lines that are greater than 79 characters are what the CI build is catching as errors. Could you reformat docstring lines that are >79 chars? One example is this one.

```suggestion
     :param str position: Specifies whether the series is moved to before or 
        after the adjacent series. Values can be either 'before' or 'after'; 
        defaults to 'before'.
```

I will mark the other lines that are causing errors.